pkgs/build-support/php/builders/v2/hooks/composer-install-hook.sh

```diff
@@ -11,9 +11,9 @@ declare -g composerNoScripts
 
 declare -ga composerFlags=()
 
-[[ 1 == "${composerNoDev:-1}" ]] && composerFlags+=(--no-dev)
-[[ 1 == "${composerNoPlugins:-1}" ]] && composerFlags+=(--no-plugins)
-[[ 1 == "${composerNoScripts:-1}" ]] && composerFlags+=(--no-scripts)
+[[ "${composerNoDev:-0}" == 1 ]] && composerFlags+=(--no-dev)
```

Comment:
Nit-picking:

People typically do `test -n` and `test -z`, which is what `$stdenv/setup.sh` is using.

```suggestion
[[ -n "${composerNoDev-}" ]] && composerFlags+=(--no-dev)
```

(I cannot do multi-line comment on mobile GitHub pages, so please help fix other lines as well.)